inference-engine/tools/vpu/vpu_perfcheck/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ function(add_perfcheck_target TARGET_NAME PLUGIN_NAME)
     target_include_directories(${TARGET_NAME}
         SYSTEM PRIVATE
             "${IE_MAIN_SOURCE_DIR}/src/vpu/graph_transformer/include"
+            "${IE_MAIN_SOURCE_DIR}/src/plugin_api"
```

Comment:
is `vpu_perfcheck` distributed as a source code?

Reply:
Now it's only used by developers and as I know, there were plans to get rid of it at all, because we have `benchmark_app` which do exactly the same and some more.